relayer/src/event.rs

```diff
@@ -1,3 +1,33 @@
+use core::fmt::Display;
+use ibc::{events::IbcEvent, Height};
+use serde::Serialize;
+
 pub mod bus;
 pub mod monitor;
 pub mod rpc;
+
+#[derive(Clone, Debug, Serialize)]
+pub struct IbcEventWithHeight {
+    pub event: IbcEvent,
+    pub height: Height,
+}
+
+impl IbcEventWithHeight {
+    pub fn new(event: IbcEvent, height: Height) -> Self {
+        Self { event, height }
+    }
+
+    pub fn event(&self) -> &IbcEvent {
```

Comment:
We don't need those if the fields are public imho

Reply:
Alternatively, we can make the the fields private.

Reply:
[ab9c8d2](https://github.com/informalsystems/ibc-rs/pull/2542/commits/ab9c8d2e3d4651b41342c22672a43ca2d60ae5f0)